blog/2021-09-11-teraspotlight-gsoc-2021-summary/index.md

```diff
@@ -0,0 +1,196 @@
+
+<div class="card w-100 mt-4 shadow p-3 mb-5 bg-white rounded" style="height: 180px">
+   <div class="row">
+   <div class="col-2"> 
+     <img  class="rounded-circle ml-4" style="height: 30%" src="https://avatars.githubusercontent.com/tolziplohu" />
+   </div>
+   <div class="col-9"> 
+     <p class="card-title text-muted">Lorxu</p>
+    <h5 class="card-text">Distinct Terrain Features for Metal Renegades</h5>
+    <hr />
+    <div>
+   <a class="text-warning" href="https://summerofcode.withgoogle.com/organizations/5338575677161472/#6733281816477696"> PROJECT CARD</a>
+    </div>
+   </div>
+   </div>
+</div>
+
+  _@Lorxu_ (aka _@tolziplohu_) completed the overhaul of the Metal Renegades world generation towards a shiny new Old West world with diverse biomes, strata, mesas, and rivers.
+  The introduction of the new biomes for scrubland, rocky terrain, rivers and steppe are accompanied by a lot of work in the background.
+
+  <div class="row">
+   <div class="col-md-6"><img src="./mr_river-real.jpg" /></div>
+   <div class="col-md-6"><img src="./mr_river-ingame.jpg" /></div>    
+  </div>
+
+  The student changed the API for biomes to be more modular and reusable and re-implemented block tinting for grass blocks and leaves.
+  More visible in-game are the more transition zones between biomes that Lorxu designed to look more messy instead of hard cut as well as the new Baobab trees based on `GrowingFlora`.
+
+  <div class="row">
+   <div class="col-md-6"><img src="./mr_terrain-transition.jpg" /></div>
+   <div class="col-md-6"><img src="./mr_baobab-trees.jpg" /></div>
+  </div>
+
+  The flat-sided mountains called "Mesas" with their characteristic layers of different types of rock called "Strata" and the new dry and water-filled rivers that meander through the world let the player immerse in the Old West setting.
+  The student proved their open-source mindset by getting active in related yet distant parts of the project, for instance the chunk ordering logic for chunk generation and processing as well as the world generation tutorial.
+
+  <div align="center">
+    <img src="./mr_mesa-comparison.jpg" width="60%" />
+  </div>
+
+  The project was mentored by _@Skaldarnar_ and _@keturn_, with additional support from _@Suhas_ (aka _@agent-q1_).
+
+<div class="card w-100 mt-4 shadow p-3 mb-5 bg-white rounded" style="height: 180px">
+   <div class="row">
+   <div class="col-2"> 
+     <img  class="rounded-circle ml-4" style="height: 30%" src="https://avatars.githubusercontent.com/ryuk156" />
+   </div>
+   <div class="col-9"> 
+     <p class="card-title text-muted">Yash Patel</p>
+    <h5 class="card-text">Migrate Web Presence to new Module Site</h5>
+    <hr />
+    <div>
+   <a class="text-warning" href="https://summerofcode.withgoogle.com/organizations/5338575677161472/#5855615500419072">PROJECT CARD</a>
+    </div>
+   </div>
+   </div>
+</div>
```

Comment:
This looks a bit off, not sure why.

![image](https://user-images.githubusercontent.com/1448874/211120257-c6413383-538b-4596-90e6-22fc2c9138b0.png)


this is how it looks on the old page

![image](https://user-images.githubusercontent.com/1448874/211120272-76bc2eff-7fe7-4805-b384-8823dd59d18d.png)

Reply:
the module site replacement of these cards is not 100% equal. for instance, clicking on the profile picture will not bring you to the contributor's github profile as it does on the current home page.
we need to fix this, but I propose we do that in a separate follow-up PR

Reply:
I think Gatsby works well with [MDX](https://mdxjs.com/), some markdown extension "for the component area" - basically a way to use custom components in markdown. See [Adding MDX Pages](https://www.gatsbyjs.com/docs/how-to/routing/mdx/) from the Gatsby documentation, for instance.